examples/generic/main.c

```diff
@@ -46,14 +46,29 @@ int main(int argc, char *argv[])
     printf("%s\n", update);
     free(update);
 
+    const char *patchBody = "[{\"op\": \"replace\", \"path\": \"/metadata/labels/foo\", \"value\": \"qux\" }]";
+    list_t *localVarHeaderType = list_create();
+    list_t *localVarContentType = list_create();
+    list_addElement(localVarHeaderType, "application/json");
+    list_addElement(localVarHeaderType, "application/yaml");
+    list_addElement(localVarHeaderType, "application/vnd.kubernetes.protobuf");
+    list_addElement(localVarContentType, "application/json-patch+json");
```

Comment:
I think this example is a little mis-leading. We should really only supply one of the patch types (e.g. `application/json-patch+json`) in the content type header (and we should drop the non-patch content types also)

Perhaps something like:

```c
// Kubernetes supports multiple content types:
// list_addElement(localVarContentType, "application/json-patch+json");
// list_addElement(localVarContentType, "application/merge-patch+json");
list_addElement(localVarContentType, "application/strategic-merge-patch+json");
// list_addElement(localVarContentType, "application/apply-patch+yaml");
```

Reply:
Thanks @brendandburns 
It's done.